src/assets/js/search.js

```diff
@@ -155,6 +148,11 @@
     return endIndex
   }
 
+  const getTermMatcher = (term, content) => {
+    const regex = new RegExp(term, 'ig')
+    return { term, regex, matchIndex: content.search(regex) }
+  }
+
```

Comment:
I think this can go above `getStartIndex` - the approach to ordering methods I was going for in this file was chronological, and `getTermMatcher` is called before the index methods

Reply:
Done :)